docs/semgrep-code/editor.md

```diff
@@ -188,6 +189,37 @@ Once you've written a rule and created comment annotations, you can run your rul
 
 <EnableTurboMode />
 
+## Code search (beta)
+
+Code search allows you to test a Semgrep rule by running it against one or more GitHub repositories or projects instead of just a few lines of test code. Its results highlight all instances of matching code in those target repositories, allowing you to see whether your rule works as intended or not. This rapid feedback can help you develop more accurate and effective rules.
+
+![Code search in Semgrep Editor](/img/code-search.png)
+***Figure.*** Code search in Semgrep Editor
+
+### Prerequisites
+
+* Code search is in **public beta** for Semgrep customers on a paid plan.
+* You must grant Semgrep code access if you would like to run code search against your repositories. Otherwise, you can run code search against public repositories. 
```

Comment:
I think this needs a rephrasing to remove "code access" -- the way to access is by going through the "Register your Github App" flow!

Reply:
The rest of this looks great though!

Reply:
So both ZCS and query console require the "Register your GitHub App" flow - I have an existing [set of instructions here](https://deploy-preview-1536--semgrep-docs-prod.netlify.app/docs/deployment/managed-scanning#install-the-private-semgrep-github-app) that have been approved by Jack & Pablo. I wonder if we want to modularize it so we can reuse it in this doc, or just link to it when it is live?